ethereum-network-partition-test/ethereum_helpers.star

```diff
@@ -5,9 +5,13 @@ LATEST_BLOCK_NUMBER_GENERIC = "latest"
 # and pad the hexadecimal string to 20 characters (which should be a limit we hopefully never hit)
```

Comment:
Update the comment as we're not padding anymore

Reply:
Done